doc/u-boot-env-salvator-x-h3-m3.txt

```diff
@@ -54,6 +54,16 @@ setenv emmc_kernel_load ext2load mmc 1:1 0x7a000000 /boot/Image
 setenv emmc_xen_load ext2load mmc 1:1 0x48080000 /boot/xen-uImage
 setenv emmc_xenpolicy_load ext2load mmc 1:1 0x7c000000 /boot/xenpolicy
 
+In case of boot SoC from the eMMC boot partition 1 (50MHz x8 bus width mode) instead of Serial Flash,
+we have an ability to flash bootloader image(s) right here in U-Boot (images must be in raw binary format):
+===================================
```

Comment:
above this line put short title, e.g. "ARMTF from eMMC"
below put full description

Reply:
ok